org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/package.json

```diff
@@ -19,12 +19,13 @@
     ]
   },
   "dependencies": {
-    "opent2t": "1.0.1",
-    "opent2t-translator-com-nest-hub": "*"
+    "opent2t": "^1.0.1",
+    "opent2t-translator-com-nest-hub": "^0.2.0"
```

Comment:
I thought we would publish new version of translators  to 0.2.0 (except Wink). I will double check with Avneet.

---
In reply to: [98255660](https://github.com/openT2T/translators/pull/146#discussion_r98255660) [](ancestors = 98255660)